src/lib/viewers/media/DashViewer.js

```diff
@@ -310,11 +310,30 @@ class DashViewer extends VideoBaseViewer {
                 break;
         }
 
+        this.showGearHDIcon(this.getActiveTrack());
+
         if (quality) {
             this.emit('qualitychange', quality);
         }
     }
 
+    /**
+     * Determines if the current track is playing HD video, then shows
+     * or hides 'HD' next to the gear icon
+     *
+     * @param {Object} activeTrack - the currently playing track
+     * @return {void}
+     */
+    showGearHDIcon(activeTrack = {}) {
```

Comment:
I know that we use do it elsewhere in this file, but our standard is normally to keep it camel case, e.g. showGearHdIcon, or isCustomUrl.